rbac/management/principal/view.py

```diff
@@ -116,8 +116,13 @@ def get(self, request):
         response_data = {}
         if status_code == status.HTTP_200_OK:
             data = resp.get('data', [])
+            if isinstance(data, dict):
```

Comment:
Will there be another followup PR? The above [proxy.request_principals](https://github.com/RedHatInsights/insights-rbac/blob/master/rbac/management/principal/view.py#L112) still using the default and calling the v1 endpoint

It would be great if a unit test is added 😁

Reply:
Made some changes, should actually use the right endpoint now, but I didn't set up a test yet. I'm not sure how to reasonably test these changes when we'd largely just be passing in the full data from the mock patch. Any suggestions?

Reply:
Right, since you have tested them with mock data, it should be good. Could test that once it is deployed in CI to verify.
I am fine with the changes. @coderbydesign Keith, do you have any suggestions?

Reply:
Since we're using the new version of the API, it's probably worth at least updating the mocked return values in all of the specs (example: https://github.com/RedHatInsights/insights-rbac/blob/master/tests/management/principal/test_view.py#L57) to reflect the v2 response?

That would test this as well in the process. We could leave one of the existing tests, and maybe add a check to ensure the `count = None` branch is executed if the payload is formatted like v1 still?

Reply:
Good idea 👍